plugins/sniff.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sniff
+spec:
+  version: "v1.0.0"
+  platforms:
+  - uri: https://github.com/eldadru/ksniff/releases/download/v1.1.0/ksniff.zip
+    sha256: afb1a2e2cd093f1c8f8fff511f48cc5a290d2c6ecd18d9f51f9c66500710297b
+    bin: kubectl-sniff
+    files:
+    - from: ./kubectl-sniff-darwin
+      to: kubectl-sniff
+    - from: ./static-tcpdump
+      to: static-tcpdump
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/eldadru/ksniff/releases/download/v1.1.0/ksniff.zip
+    sha256: afb1a2e2cd093f1c8f8fff511f48cc5a290d2c6ecd18d9f51f9c66500710297b
+    bin: kubectl-sniff
+    files:
+    - from: ./kubectl-sniff
+      to: kubectl-sniff
+    - from: ./static-tcpdump
+      to: static-tcpdump
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/eldadru/ksniff/releases/download/v1.1.0/ksniff.zip
+    sha256: afb1a2e2cd093f1c8f8fff511f48cc5a290d2c6ecd18d9f51f9c66500710297b
+    bin: kubectl-sniff.exe
+    files:
+    - from: ./kubectl-sniff-windows
+      to: kubectl-sniff.exe
+    - from: ./static-tcpdump
+      to: static-tcpdump
```

Comment:
This probably should be an exe

Reply:
static-tcpdump is always a ELF, it's not executed locally but in a remote container running on kubernetes.